document/core/exec/instructions.rst

```diff
@@ -964,24 +964,22 @@ Exiting :math:`\instr^\ast` with label :math:`L`
 
 When the end of a block is reached without a jump or trap aborting it, then the following steps are performed.
 
-1. Let :math:`n` be the arity of :math:`L`.
+1. Let :math:`m` be the number of values on the top of the stack.
```

Comment:
I know what you mean by "values on top of the stack", but is that mentioned anywhere in the spec?

Reply:
The runtime section defines the stack. Although never defined explicitly, the spec mentions "the top of the stack" in various places. Do you think it might be too unclear?

Reply:
I think it's probably fine when it's just a few values. This case is perhaps a little less clear since you're only counting up from the label. That said, the formula below makes it explicit.